python_modules/dagster/dagster/_annotations.py

```diff
@@ -815,3 +815,10 @@ def only_allow_hidden_params_in_kwargs(annotatable: Annotatable, kwargs: Mapping
             deprecated_params[param].hidden,
             f"Unexpected non-hidden deprecated parameter '{param}' in kwargs. Should never get here.",
         )
+
+
+# placeholder
+beta = experimental
+beta_param = experimental_param
+alpha = experimental
```

Comment:
should we call it "Preview" to match the api language in the [notion proposal](https://www.notion.so/dagster/API-Lifecycle-Refresh-f72f6e199da3409c8c5da297a109b3c8)?